src/lib/Components/Inbox/Bids/ActiveBid.tsx

```diff
@@ -0,0 +1,165 @@
+
+import colors from "../../../../data/colors"
+import OpaqueImageView from "../../OpaqueImageView"
+
+const Container = styled.View`
+  margin: 17px 20px 0px;
+`
+
+const Content = styled.View`
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`
+
+const ImageView = styled(OpaqueImageView)`
+  width: 58px;
+  height: 58px;
+  border-radius: 4;
+`
+
+const MetadataContainer = styled.View`
+  margin: 0 15px;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Separator = styled.View`
+  height: 1;
+  width: 100%;
+  background-color: ${colors["gray-regular"]};
+  margin-top: 17px;
+`
+
+const StatusLabel = styled(MetadataText)`
+  color: ${(props: { status: BidStatus }) => {
+    switch (props.status) {
+      case "winning":
+        return colors["green-regular"]
+      case "reserve":
+        return colors["yellow-regular"]
+      case "losing":
+        return colors["red-regular"]
+    }
+  }};
+`
+
+type BidStatus = "winning" | "reserve" | "losing"
+
+interface State {
+  status: BidStatus
+}
+
+class ActiveBid extends React.Component<any, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      status: "losing",
+    }
+  }
+
+  componentDidMount() {
+    this.updateStatus()
+  }
+
+  updateStatus() {
+    const bid = this.props.bid
+    const leadingBidder = bid.is_leading_bidder
+    const reserveNotMet = bid.active_bid.sale_artwork.reserve_status === "reserve_not_met"
+
+    let status: BidStatus = "losing"
+    if (leadingBidder) {
+      status = reserveNotMet ? "reserve" : "winning"
+    }
+
+    this.setState({ status })
+  }
+
+  get statusLabel(): string {
+    switch (this.state.status) {
+      case "winning":
+      case "reserve":
+        return "Highest Bid"
+      case "losing":
+        return "Outbid"
+    }
+  }
+
+  render() {
+    const bid = this.props.bid.active_bid
+    const imageURL = bid.sale_artwork.artwork.image.url
+    const lotNumber = bid.sale_artwork.lot_number
+    const artistName = bid.sale_artwork.artwork.artist_names
+
+    const headline = `Lot ${lotNumber} · ${artistName} `
+    const subtitle = `Current Bid: ${bid.max_bid.display} `
+
+    return (
+      <TouchableWithoutFeedback>
+        <Container>
+          <Content>
+            <ImageView imageURL={imageURL} />
+            <MetadataContainer>
+              <BodyText>{headline}</BodyText>
+              <BodyText>{subtitle}</BodyText>
+            </MetadataContainer>
+            <StatusLabel status={this.state.status}>{this.statusLabel}</StatusLabel>
+          </Content>
+          <Separator />
+        </Container>
+      </TouchableWithoutFeedback>
+    )
+  }
+}
+
+export default Relay.createContainer(ActiveBid, {
+  fragments: {
+    bid: () => Relay.QL`
+      fragment on LotStanding {
+        is_leading_bidder
+        active_bid {
+          id
+          max_bid {
+            cents
+            display
+          }
+          sale_artwork {
```

Comment:
It looks like we're still using `sale_artwork` within the `active_bid` instead of within the lot standing directly – the [`active_bid`](https://github.com/artsy/metaphysics/blob/master/schema/me/lot_standing.js#L17-L20) field is only populated if the logged-in user is winning. I think we want to use the [`sale_artwork` on the lot standing itself](https://github.com/artsy/metaphysics/blob/master/schema/me/lot_standing.js#L40-L42) instead. Let me know if I can clarify anything.